package.json

```diff
@@ -77,6 +77,10 @@
     "react-svg-loader": "^2.1.0",
     "react-virtualized-select": "^3.1.3",
     "redux-devtools-extension": "^2.13.7",
+    "reg-notify-github-plugin": "^0.7.19",
```

Comment:
maybe pin these exactly? could help with security vulnerabilities in the future?